.github/workflows/ci.yml

```diff
@@ -12,24 +12,24 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [18.x]
+        node-version: [20.x]
     steps:
       - uses: actions/checkout@v3
-      - uses: ./.github/actions/pnpm
+      - uses: ./.github/actions/env
         with:
           node-version: ${{ matrix.node-version }}
       - name: Test
         run: pnpm test
```

Comment:
should be `bun run test`

Reply:
Fixed. It's interesting to note `bun test` gives a different test suite result. For anyone who might be confused initially in the future.